.github/workflows/main.yml

```
@@ -1,106 +1,84 @@
-  DOCKER_VERSION: '233.14938'
-
 jobs:
-  deploy:
+  checkout:
     runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get_version.outputs.VERSION }}
     steps:
-      - name: Checkout
+      - name: Checkout code
         uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)